### ISSUE TITLE
Documentation for rank_genes_groups 

### ISSUE THREAD

**Issue by sinclaircooper** [scverse/scanpy#446]
Hi, 

Thanks for the great tools included in scanpy. 

I was searching to see what the logfoldchange numbers in the `rank_genes_groups` are. i.e. is it natural log/log base 10 or log2? The code in https://github.com/theislab/scanpy/blob/master/scanpy/tools/_rank_genes_groups.py shows that it is log2foldchange. 

Perhaps this should be made clear in the documentation? Or `logfoldchange` should be chnaged to `log2foldchange`. 

Cheers, 
S

**Comment by falexwolf** [comment]
Thank you for the kind words!

Hm, it's in the [docs in the returns section](https://icb-scanpy.readthedocs-hosted.com/en/latest/api/scanpy.api.tl.rank_genes_groups.html). Should it be somewhere else?

But you're right, I'm also not completely happy with the name `logfoldchange`. We'll harmonize with what's out there at some point and I'm currently tending to `log2FC` because @davidsebfischer started to using that in `diffxpy` and I guess it's also used in some R tools.

**Comment by sinclaircooper** [comment]
Whoops, sorry, I read that returns section a couple of times but somehow missed the `log2` bit! Apologies. 

However, I agree with you that renaming the `log2FC` would remove any ambiguity.

**Comment by davidsebfischer** [comment]
@falexwolf, yes I think overall fold changes with 2 as a basis are more intuitive than with e as a basis for most people, similar to basis=10, but basis 2 gives a more sensible dynamic range than 10 does on gene expression data.

**Comment by falexwolf** [comment]
Thanks David, I completely agree, but I was really just talking about the naming convention: `log2FC` vs. `log2foldchange` vs. `logfoldchange` vs. other possibilities... 🙂

**Comment by a-munoz-rojas** [comment]
Hi all - I just saw this and I agree that `log2FC` is a better name. I would be happy to help with this small change if necessary!
